ONSdigital/eq-author
# Initial creation of superuser

### Changes

Creates an initial superuser account based on environment variables provided by terraform

### How to test
1) Ensure the environment variables are present within the elasticbeanstalk environment
2) Checkout this branch
3) Run eb deploy
4) Visit `/admin` on the newly deployed application
5) Login with the credentials supplied by terraform
6) ???
7) Profit!!!!

### Who can test

Anyone except @weapdiv-david
